test/e2e/cloud/gcp/cluster_upgrade.go

```diff
@@ -89,26 +89,26 @@ var kubeProxyDowngradeTests = []upgrades.Test{
 	&upgrades.ServiceUpgradeTest{},
 }
 
-// masterUpgrade upgrades master node on GCE/GKE.
-func masterUpgrade(f *framework.Framework, v string) error {
+// instanceUpgrade upgrades instance node on GCE/GKE.
+func instanceUpgrade(f *framework.Framework, v string) error {
```

Comment:
This technically refer to an "instance" of the control plane (since control plane can span multiple machines/VMs in HA environments) so something like `controlPlaneInstance` would be more preferable if we want to tag on the control plane prefix.